playbooks/loki.yaml

```diff
@@ -1,9 +1,12 @@
 # vim:ts=2:sw=2:et:ai:sts=2
-
-# Playbook: Configure loki.
 ---
-- name: Configure loki
+# =============================================================================
+# Install and configure loki log collector and analyser.
+# =============================================================================
+
+- name: Install and configure loki log collector and analyser
   hosts: loki_servers
```

Comment:
This one wasn't updated to take `target_hosts`, not sure if intentionally or not. (Same for promtail, but gets deleted in a later PR.)

Reply:
Ah; _also_ deleted in a _different_ PR!